tests/test_movingtpf.py

```diff
@@ -205,3 +205,29 @@ def test_make_tpf():
 
     # Delete the file
     os.remove("tests/tess-1998YT6-s0006-shape11x11-moving_tp.fits")
+
+
+def test_ellipse_aperture():
+    """
+    Test ellipse aperture. The ellipse is fitted by computing the first- and second-order
```

Comment:
```suggestion
    Test ellipse aperture. The ellipse is calculated by computing the first- and second-order
```

Reply:
Not sure it's right to say fitted when the ellipse parameters are calculated from moments.